pyiode/objects/table.pyx

```diff
@@ -22,8 +22,14 @@ cdef class TableCell:
 
     Attributes
     ----------
+    # GDM>
+    # * I *wonder* if using StringCell and LecCell subclasses wouldn't be better (for example if LecCell had additional methods)?
+    # * either the type hint or possible values are wrong.
     cell_type: int
         Type of cell content. Possible value is 'STRING' or 'LEC'.
+    # GDM>
+    # * unsure allowing both string and ints brings anything. Should use either Enum or strings IMO.
```

Comment:
*align* parameter became an Enum (as many parameters of methods of classes from the table.pyx file) -> see PR #574